src/client/components/VesselsList.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import Vessels from "./VesselTeaser";
```

Comment:
import it under the name `VesselTeaser` for consistency.

Reply:
renamed VesselTeaser